src/com/MPICommunication.cpp

```diff
@@ -83,26 +83,26 @@ PtrRequest MPICommunication::aSend(const int *itemsToSend, int size, Rank rankRe
   return PtrRequest(new MPIRequest(request));
 }
 
-void MPICommunication::send(const double *itemsToSend, int size, Rank rankReceiver)
+void MPICommunication::send(precice::span<const double> itemsToSend, Rank rankReceiver)
```

Comment:
Do we need this code duplication of `void MPICommunication::send(precice::span<const int> itemsToSend, Rank rankReceiver)`?

Reply:
We could avoid it, but the alternatives are more verbose and complex (templating, partial specialisation etc)